test/functional/Jsr292/src/com/ibm/j9/jsr292/indyn/IndyTest.java

```diff
@@ -370,4 +370,36 @@ public void test_boostrap_return_constant_MethodType() {
 		AssertJUnit.assertEquals(expected, mt);
 		AssertJUnit.assertTrue(expected == mt);
 	}
+
+	// test that if resolved CallSite is null, the same error is rethrown
+	@Test(groups = { "level.extended" })
+	public void test_CallSiteNullErrorRethrown () {
+		// BSME is expected on the first run
+		boolean bootstrapMethodErrorThrown1 = false;
+		try {
+			com.ibm.j9.jsr292.indyn.GenIndyn.test_CallSiteNullErrorRethrown();
```

Comment:
Add an explicit fail after the call if an exception is expected
```suggestion
			com.ibm.j9.jsr292.indyn.GenIndyn.test_CallSiteNullErrorRethrown();
			Assert.fail("Expected a BootstrapMethodError to be thrown");
```